frontend/src/app/modules/availability/edit-slot/edit-availability-page/edit-availability-page.component.ts

```diff
@@ -77,9 +78,13 @@ export class EditAvailabilityPageComponent extends BaseComponent implements OnDe
             hasAdvancedSettings: this.newAvailabilityComponent.generalComponent.addAdvanced,
             advancedSettings,
             eventDetails: this.newAvailabilityComponent.eventDetailComponent.settings,
-            questions: this.newAvailabilityComponent.questionsComponent.questions,
             schedule: this.newAvailabilityComponent.scheduleComponent.schedule,
             templateSettings: this.newAvailabilityComponent.notificationEmailsComponent.settings,
+            questions: this.newAvailabilityComponent.questionsComponent.questions.map((question) => {
+                question.questionText = removeExcessiveSpaces(question.questionText);
+
+                return question;
+            }),
```

Comment:
```suggestion
            questions: this.newAvailabilityComponent.questionsComponent.questions,
```
The _questions component_ should be responsible for correct processing of user input and presenting it in usable form.